iothub/device/src/Common/Exceptions/IotHubException.cs

```diff
@@ -18,41 +18,15 @@ public class IotHubException : Exception
         [NonSerialized]
         private const string TrackingIdValueSerializationStoreName = "IotHubException-TrackingId";
 
-        [NonSerialized]
-        private bool _isTransient;
-
-        [NonSerialized]
-        private string _trackingId;
-
         /// <summary>
         /// Gets a value indicating if the error is transient.
         /// </summary>
-        public bool IsTransient
-        {
-            get
-            {
-                return _isTransient;
-            }
-            private set
```

Comment:
changing from private set to set. Intentional?

Reply:
no, good catch